Task10/App/Services/YoutubeService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using App.Entities;
+using Newtonsoft.Json.Linq;
+using static App.Controllers.YoutubeController;
+
+namespace App.Services
+{
+    public class YoutubeService
+    {
+        private readonly string API_KEY = "AIzaSyALMiCr_df0KxwdiWxpGeQBSCrczG5RcRs";
+
+        public async Task<YoutubeResponse> GetFromApi(string query)
+        {
+            HttpClient client = new HttpClient();
+            var videoList = new List<YoutubeItem>();
+            try
+            {
+                HttpResponseMessage response = await client.GetAsync($"https://www.googleapis.com/youtube/v3/search?part=snippet&key={this.API_KEY}&{query}");
```

Comment:
Ideally this url and api token should be stored in appsettings

Reply:
Done